dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -1036,7 +1036,7 @@ public Entry getPublishedEntryByPath(@ApiParam(value = "repository path", requir
     @UnitOfWork(readOnly = true)
     @Path("/path/workflow/{repository}/published")
     @ApiOperation(nickname = "getPublishedWorkflowByPath", value = "Get a published workflow by path", notes = "Does not require workflow name.", response = Workflow.class)
-    public Workflow getPublishedWorkflowByPath(@ApiParam(value = "repository path", required = true) @PathParam("repository") String path, @ApiParam(value = "Comma-delimited list of fields to include: validations") @QueryParam("include") String include,
+    public Workflow getPublishedWorkflowByPath(@ApiParam(value = "repository path", required = true) @PathParam("repository") String path, @ApiParam(value = "Comma-delimited list of fields to include: validations, aliases") @QueryParam("include") String include,
```

Comment:
Use a constant string for "aliases" and use it both here and in the (now badly named) `initializeValidations` method. 
That way, someone won't accidentally rename one but not the other. 
Should also do the same for validations while in the neighbourhood.

Reply:
Done